pkgs/applications/misc/flet-client-flutter/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, fetchFromGitHub
+, pkg-config
+, python3Packages
+, flutter
+, gst_all_1
+, libunwind
+, makeWrapper
+, orc
+}:
+
+flutter.buildFlutterApplication rec {
+  pname = "flet-client-flutter";
+  inherit (python3Packages.flet) version;
+
+  sourceRoot = "source/client"; # the flutter code is not on the root of the repo
+
+  src = fetchFromGitHub {
+    owner = "flet-dev";
+    repo = "flet";
+    rev = "v${version}";
+    hash = { # avoid stale hashes after a update
+      "0.7.4" = "sha256-HUT3+Pvys3VI5oHnCTe2JTUyusq8t2890cj2pbf3e6s=";
+    }.${version} or "sha256:${lib.fakeSha256}";
```

Comment:
Please don't, if this is an issue for you use something like `nix-update` that will properly update the version and hash at the same time.

Reply:
In this case it's important that the client bumps together with the python module so I am using the python module version to get which revision will be used to build the client. This would only be a reminder to update the hash of the client too when bumping.

Reply:
Don't make a package to be dependent on another one version since this tends to cause a multitude of other issues.

Better to keep both of them separate and add a note for the maintainer to update both at the same time.

Reply:
Solved

I am now using the src from the flutter program as base. When the flutter program is updated both python libraries are updated by consequence.

Reply:
And I think I broke the patch lol